packages/api/package.json

```diff
@@ -12,8 +12,8 @@
   "scripts": {
     "cloud-dev": "npm run docker && kubectl delete pod -l app.kubernetes.io/name=livepeer-api-server --wait && sleep 1 && kubectl logs -f --pod-running-timeout=30s $(kubectl get pods -o name -l app.kubernetes.io/name=livepeer-api-server)",
     "cloud-dev:server": "nodemon -w src -x node -r esm src/cli.js --port 3040 --kubeBroadcasterService broadcaster --kubeOrchestratorService orchestrator --kubeNamespace default --kubeBroadcasterTemplate 'https://{{nodeName}}.livepeer-staging.live' --kubeOrchestratorTemplate 'https://{{nodeName}}.livepeer-staging.live:8935'",
-    "docker:build": "docker pull livepeerci/api:master && docker build --cache-from=livepeerci/api:master -t livepeerci/api:${DOCKER_TAG:-master} --build-arg GITHUB_SHA --build-arg DOCKER_TAG -f Dockerfile ../.. && docker run livepeerci/api:${DOCKER_TAG:-master} --help",
-    "docker:push": "docker push livepeerci/api:${DOCKER_TAG:-dev}",
+    "docker:build": "TAGS=\"${DOCKER_TAGS:-dev}\"; docker pull livepeerci/api:master && docker build --cache-from=livepeerci/api:master $(printf ' -t livepeerci/api:%s' $TAGS) --build-arg GITHUB_SHA --build-arg VERSION -f Dockerfile ../.. && docker run livepeerci/api:$(printf $TAGS)  --help",
```

Comment:
I use the `master` build here for building new things for livepeer-in-a-box. Any idea the best way to handle that workflow?

Reply:
Humm, not really sure. Would it be possible to use the `dev` tag on `livepeer-in-a-box` instead? If not easily, I can just revert that specific commit that did this: https://github.com/livepeer/livepeer-com/pull/625/commits/0c881f725b79c9db7ee69b7c056c560a77f249bb
I don't think it is a must.

Reply:
Reverted but kept the default `push` tag different than the build so we don't accidentally push to the production image tag.